jmh-benchmarks/src/main/java/org/apache/kafka/jmh/assignor/ServerSideAssignorBenchmark.java

```diff
@@ -263,13 +270,36 @@ private void simulateIncrementalRebalance() {
             Collections.emptyMap()
         ));
 
-        assignmentSpec = new AssignmentSpec(updatedMembers, subscriptionType);
+        groupSpec = new GroupSpecImpl(updatedMembers, subscriptionType, invertedTargetAssignment);
+    }
+
+    private Map<Uuid, Map<Integer, String>> updateInvertedTargetAssignment(
+        GroupAssignment targetAssignment
+    ) {
+        Map<Uuid, Map<Integer, String>> invertedTargetAssignment = new HashMap<>(topicCount);
+        for (Uuid topicId : allTopicIds) {
+            invertedTargetAssignment.put(topicId, new HashMap<>(subscribedTopicDescriber.numPartitions(topicId)));
+        }
```

Comment:
This is not needed. I think that we can init it by using `computeIfAbsent` like you did in `invertedTargetAssignment`. With this change, the two methods are identical so we can share the code.